test/e2e/apps/types.go

```diff
@@ -35,4 +35,7 @@ var (
 
 	// AgnhostImage is the fully qualified URI to the Agnhost image
 	AgnhostImage = imageutils.GetE2EImage(imageutils.Agnhost)
+
+	// InvalidImage is a fully qualified URI pointing to an unexisting image
```

Comment:
non-existent? (instead of unexisting)

Reply:
changed in 6985b3543feb46d6819c16c6babf6f08733e90cf